sdk/deepbook/src/deepbook_sdk.ts

```diff
@@ -69,11 +57,11 @@ export class DeepBook_sdk {
 	 * @description: Create and Transfer custodian account to user
 	 * @param currentAddress: current user address, eg: "0xbddc9d4961b46a130c2e1f38585bbc6fa8077ce54bcb206b26874ac08d607966"
 	 */
-	async createAccount(currentAddress: string): Promise<TransactionBlock | TransactionArgument> {
+	public createAccount(currentAddress: string): TransactionBlock {
 		const txb = new TransactionBlock();
 		let [cap] = txb.moveCall({
 			typeArguments: [],
-			target: `dee9::clob::create_account`,
+			target: `dee9::clob_v2::create_account`,
```

Comment:
Rather than repeat this every time, and having us potentially miss this each time we update the clob, can we extract this to a constant file somewhere and read in the constant?